playgrounds/app/src/components/Editor.tsx

```diff
@@ -338,23 +352,101 @@ export default function Editor(props: EditorProps) {
                     <TbSettings size={24} />
                   </Button>
                 </DropdownMenuTrigger>
-                <DropdownMenuPortal>
-                  <DropdownMenuContent class="w-[160px]">
-                    <DropdownMenuItem
-                      class="flex flex-row items-center justify-between"
-                      closeOnSelect={false}
-                    >
-                      <Label for="bg-color-input" class="font-normal">
-                        BG Color
-                      </Label>
-                      <input
-                        id="bg-color-input"
-                        class="h-6 w-6 rounded"
-                        type="color"
-                        value={props.bgColor}
-                        onInput={e => props.setBgColor(e.target.value)}
-                      />
-                    </DropdownMenuItem>
+								<DropdownMenuPortal>
```

Comment:
The formatting here seems pretty wonky. Make sure you are running the autoformatting.